src/server/game/Entities/Object/Object.cpp

```diff
@@ -1500,10 +1500,10 @@ float WorldObject::GetGridActivationRange() const
     {
         if (ToPlayer()->GetCinematicMgr()->IsOnCinematic())
             return DEFAULT_VISIBILITY_INSTANCE;
-        return GetMap()->GetVisibilityRange();
+        return GetMap()->GetVisibilityRange() * 5;
```

Comment:
Not yet. Haven't been able to find a good way to update a specific creature without having to keep the whole grid loaded.